std/assembly/map.ts

```diff
@@ -88,8 +88,9 @@ export class Map<K,V> {
       changetype<usize>(this.buckets) + <usize>(hashCode & this.bucketsMask) * BUCKET_SIZE
     );
     while (entry) {
-      if (!(entry.taggedNext & EMPTY) && entry.key == key) return entry;
-      entry = changetype<MapEntry<K,V>>(entry.taggedNext & ~EMPTY);
+      let next = entry.taggedNext;
```

Comment:
Perhaps let's name the variable `taggedNext` to not confuse it with a `next` pointer.

Reply:
Done!